src/System.Device.Gpio/System/Device/Pwm/Channels/Windows10PwmChannel.Windows.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Threading;
+
+namespace System.Device.Pwm
+{
+    internal class Windows10PwmChannel : PwmChannel
```

Comment:
This is just a placeholder for now.  Supporting code will be added after Linux flavor has been determined.

Reply:
not sure I follow

Reply:
> not sure I follow

Mainly just a note-to-self and is far I got over the weekend.  Trying to setup Win10 on another RPi and also wanted to make sure the new structure looked okay before porting code for this flavor.